manifests/ovs-cni.yml.in

```diff
@@ -36,7 +37,78 @@ spec:
         volumeMounts:
         - name: cnibin
           mountPath: /host/opt/cni/bin
+      - name: ovs-cni-marker
+        image: ${OVS_CNI_MARKER_IMAGE_REPO}/${OVS_CNI_MARKER_IMAGE_NAME}:${OVS_CNI_MARKER_IMAGE_VERSION}
+        imagePullPolicy: ${OVS_CNI_MARKER_IMAGE_PULL_POLICY}
+        securityContext:
+          privileged: true
+        command:
+          - sh
+          - -c
+          - |
+            if [ -e /host/var/run/openvswitch/db.sock ]; then
```

Comment:
this should be hidden under an entrypoint script

Reply:
I tried to split it, so image is universal and manifests handles all deployment gaps (in this case it works with both host ovs and our opeshift stolen ovs)

Reply:
I just don't like that this logic is inside manifest. I am fine with keeping the same if-else decision making for both images. (hostMount is a separate issue.)

Reply:
Solved using separate openshift/kubernetes manifests

Reply:
I am also thinking out load here: "Why can't we just mount a subdirectory into the container and not care about host path? Then we could get rid of if-else since the socket file would be in the same well known path.

Reply:
We cannot dynamically decide which hostDirectory in yaml specification

Reply:
What do you mean? Can't you just always mount the directory, regardless where it's exposed on host, into the same directory?

Reply:
Aha like this, yea, if we have one manifest for kubernetes and one for openshift we could